constraints_config.toml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Abstract Machines
+# SPDX-License-Identifier: Apache-2.0
+
+[things]
+[things.limits]
+create = 10
+update = 10
+
+[users]
+[users.limits]
+create = 10
+update = 10
+
+[auth]
+[auth.limits]
+create = 10
+update = 10
+
+[channels]
+[channels.limits]
+create = 10
+update = 10
+
+[groups]
+[groups.limits]
+create = 10
+update = 10
```

Comment:
May be we should switch to YAML file

Reply:
YAML file should have default config, which apply for all domain, 
And it should have domain specific constraint, it will be applied for specific domain, 
If domain specific constraint is not available then it should consider for default config